### PR TITLE
Base Quest Anniversary NFT mint

### PR DESCRIPTION
**Pull request by zencephalon**
**What changed? Why?**

Nav headings, connect wallet functionality, new mint page. All to support this NFT launch.

Screenshot 2024-04-25 at 12 51 38 PM

Screenshot 2024-04-25 at 12 54 26 PM

**Notes to reviewers**

**How has it been tested?**

Extensively via manual testing and bug bash

**Does this PR add a new token to the bridge?**

Are you adding an entry to [`assets.ts`](../apps/bridge/assets.ts)?

- [x] No, this PR does not add a new token to the bridge
- [ ] Yes, and I've confirmed this token doesn't use a bridge override
- [ ] Yes, and I've confirmed this token is an OptimismMintableERC20

If you are adding a token to the bridge, please include evidence of both confirmations above for your reviewers.

### CODE REVIEW

**Review by wbnns — APPROVED** [review]

File: apps/web/next.config.js [hunk 1]
```diff
@@ -56,21 +56,41 @@ const baseXYZDomains = 'https://base.mirror.xyz';
...
+const walletconnectDomains =
+  'https://*.walletconnect.org wss://*.walletconnect.org wss://*.walletconnect.com https://*.walletconnect.com https://explorer-api.walletconnect.com';
 
 const contentSecurityPolicy = {
   'default-src': [
     "'self'",
     "'unsafe-inline'", // NextJS requires 'unsafe-inline'
     isLocalDevelopment ? "'unsafe-eval'" : '',
     baseXYZDomains,
-    greenhouseDomains,
```
Comment (lukasrosario) [inline apps/web/next.config.js:67]: does this break greenhouse? iirc we need to embed some things

Comment (zencephalon) [reply apps/web/next.config.js:67]: Based on my testing it seems sufficient to have it `connect-src` ie jobs still show up with this config

File: apps/web/pages/_app.tsx [hunk 2]
```diff
@@ -16,6 +22,15 @@ import ClientAnalyticsScript from '../src/components/ClientAnalyticsScript/Clien
 import { cookieManagerConfig } from '../src/utils/cookieManagerConfig';
 import { useSprig } from 'apps/web/src/utils/hooks/useSprig';
 
+const config = getDefaultConfig({
+  appName: 'Base.org',
+  projectId:
+    process.env.NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID ?? '99a7649b4421d35d175fbca94c1bd880',
```
Comment (lukasrosario) [inline apps/web/pages/_app.tsx:28]: do we need the hardcoded id here?

File: apps/web/public/.well_known/walletconnect.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+074c7530-876a-4af0-98ca-b084e3586fb5=32eb4d878d84c69d3d9d3aaf789e24dd0a2317a861e255e95268dc3dab06fb75
```
Comment (lukasrosario) [inline apps/web/public/.well_known/walletconnect.txt:1]: what is this?

File: apps/web/src/components/BuilderNft/BuilderNftHero.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  SubHeadingForStatus,
+} from 'apps/web/src/components/BuilderNft/constants';
+import { NftButton } from 'apps/web/src/components/BuilderNft/NftButton';
+import { FaqContent } from 'apps/web/src/components/BuilderNft/FaqContent';
+
+export function BuilderNftHero() {
+  const mintState = useMintState();
+  const { status } = mintState;
+
+  console.log({ status });
```
Comment (lukasrosario) [inline apps/web/src/components/BuilderNft/BuilderNftHero.tsx:17]: remove

File: apps/web/pages/_app.tsx (hunk 2 above)
Comment (zencephalon) [reply apps/web/pages/_app.tsx:28]: Build kite seems to break without it even though we have the ENV var set

File: apps/web/public/.well_known/walletconnect.txt (hunk 3 above)
Comment (zencephalon) [reply apps/web/public/.well_known/walletconnect.txt:1]: It's for wallet connect domain verification

File: apps/web/src/components/BuilderNft/useMintState.tsx [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+    }
+  }, [isSuccess, address]);
+  useEffect(() => {
+    if (proofQuery.isError) {
+      logEvent('builder_nft_ineligible', { address });
+    }
+  }, [proofQuery.isError, address]);
+  useEffect(() => {
+    if (error) {
+      logEvent('builder_nft_mint_error', { address });
```
Comment (lukasrosario) [inline apps/web/src/components/BuilderNft/useMintState.tsx:109]: nice 🙏

**Review by lukasrosario — DISMISSED** [review]

**Review by dneilroth — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for dneilroth @ 2024-04-25 21:37:24 UTC
User must have write permissions to review

**Review by wbnns — APPROVED** [review]

**Pull request merged**